### PR TITLE
fix: clean up permissions errors

### PR DESCRIPTION
**Pull request by eschutho**
### SUMMARY

Testing to fix the ephemeral builds
### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by mistercrunch — APPROVED** [review]

**Comment by eschutho** [comment]
That first commit worked but not the last one. I need to tweak the env var directory lookup a bit.

**Comment by eschutho** [comment]
This version is working now if we want to merge. I can iterate on finding a way to use the environment variables so that we don't need to hard code the app path.

File: Dockerfile [hunk 1]
```diff
@@ -121,6 +121,8 @@ RUN --mount=type=bind,source=./docker,target=/docker \
         libecpg-dev \
         libldap2-dev \
     && touch superset/static/version_info.json \
+    && chown -R superset:superset /app \
+    && chmod -R 775 /app \
```
Comment (mistercrunch) [inline Dockerfile:125]: wondering if these 2 commands would be belong right before the `USER superset` command, and if we do so we might be able to simplify the `COPY` commands and other chown/chmod throughout the Dockerfile

Comment (mistercrunch) [reply Dockerfile:125]: https://github.com/apache/superset/pull/31385

**Pull request closed without merging**